apps/test/unit/templates/teacherDashboard/EmptySectionsV1Test.jsx

```diff
@@ -2,7 +2,7 @@ import {render, screen} from '@testing-library/react';
 import React from 'react';
 import {BrowserRouter as Router} from 'react-router-dom';
 
-import EmptySectionV1 from '@cdo/apps/templates/teacherDashboard/EmptySectionV1';
+import {UnconnectedEmptySectionV1 as EmptySectionV1} from '@cdo/apps/templates/teacherDashboard/EmptySectionV1';
```

Comment:
Completely optional nit: I personally like importing `{UnconnectedEmptySectionV1}` as itself so that when I'm just looking at the test and not the imports I know that it is not connected to redux.